README.md

```diff
@@ -1,39 +1,431 @@
+
+#! Usage from Daru::IO
+Daru::IO::Exporters::Excel.new(df, args).call
+
+#! Usage from Daru::DataFrame
+df.to_excel(args)
+```
+
+- Have a look at [this documentation](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/Excel) to know more about this Exporter.
+- Dependency : spreadsheet gem.
+
+### JSON Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+```ruby
+#! Partially require just JSON Exporter
+require 'daru/io/exporters/json'
+
+#! Usage from Daru::IO
+Daru::IO::Exporters::JSON.new(df, args).call
+
+#! Usage from Daru::DataFrame
+df.to_json(args)
+```
+
+- Have a look at [this documentation](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/JSON) to know more about this Exporter.
+- Dependency : Standard library JSON gem and jsonpath gem.
+
+### RData Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+```ruby
+#! Partially require just RData Exporter
+require 'daru/io/exporters/r_data'
+
+#! Usage from Daru::IO
+Daru::IO::Exporters::RData.new(args).call
+```
+
+- Have a look at [this documentation](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/RData) to know more about this Exporter.
+- Dependency : rsruby gem and R.
+
+### RDS Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+```ruby
+#! Partially require just RDS Exporter
+require 'daru/io/exporters/rds'
+
+#! Usage from Daru::IO
+Daru::IO::Exporters::RDS.new(df, args).call
+
+#! Usage from Daru::DataFrame
+df.to_rds(args)
+```
+
+- Have a look at [this documentation](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/RDS) to know more about this Exporter.
+- Dependency : rsruby gem and R.
+
+### SQL Exporter
+
+[(Go to Table of Contents)](#table-of-contents)
+
+```ruby
+#! Partially require just SQL Exporter
+require 'daru/io/exporters/sql'
+
+#! Usage from Daru::IO
+Daru::IO::Exporters::SQL.new(df, args).call
+
+#! Usage from Daru::DataFrame
+df.to_sql(args)
+```
 
-## Development
+- Have a look at [this documentation](http://www.rubydoc.info/github/athityakumar/daru-io/master/Daru/IO/Exporters/SQL) to know more about this Exporter.
+- Dependency : dbd-sqlite3, dbi and sqlite3 gems.
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+# Contributing
 
-To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
+[(Go to Table of Contents)](#table-of-contents)
 
-## Contributing
+Contributions are always welcome. Please have a look at the [contribution guidelines](CONTRIBUTING.md) first. :tada:
```

Comment:
I believe, that the section "creating your own importer/exporter" (as a separate lib/gem) is necessary. As well as (at the very beginning of README) a statement that it is not only a bunch of them, but also the "framework for easy creation of new importers/exporters".